src/client/eppo-client.ts

```diff
@@ -661,9 +699,6 @@ export default class EppoClient {
       metaData: this.buildLoggerMetadata(),
       evaluationDetails,
     };
-    this.logBanditAction(banditEvent);
```

Comment:
logging has been moved up to getBanditActionDetails so more details can be returned from this method and pushes side effects out of `evaluateBandit`